plugins/vli/fu-vli-pd-parade-device.c

```diff
@@ -0,0 +1,689 @@
+/*
+ * Copyright (C) 2015-2019 VIA Corporation
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-chunk.h"
+
+#include "fu-vli-pd-device.h"
+#include "fu-vli-pd-parade-device.h"
+
+struct _FuVliPdParadeDevice
+{
+	FuDevice		 parent_instance;
+	FuVliDeviceKind		 device_kind;
+	guint8			 page_addr;
+	guint8			 page7_addr;
```

Comment:
Are there better names you can use than page and page7?

Reply:
I'm not quite sure what they are myself. The original code was somewhat, shall we say, _creative_. I'll do some research.

Reply:
Okay, it's the start address of page2 and page7 on the device itself. I've renamed and added some comments.